config/protocol.mainnet.yml

```diff
@@ -82,17 +82,21 @@ ApplicationConfiguration:
 
   RPC:
     Enabled: true
+    Addresses:
+      - ":10332"
     MaxGasInvoke: 15
     EnableCORSWorkaround: false
-    Port: 10332
     TLSConfig:
       Enabled: false
-      Port: 10331
+      Addresses:
```

Comment:
I believe this doesn't make much sense, as it's hard to imagine a case, when I'd like to use the same tls certificate for multiple addresses.

Could you re-work it the same way as neofs does?


https://github.com/nspcc-dev/neofs-node/blob/f3e59194744acc814c7ce0b4da8793749e31dede/config/example/node.yaml#L48-L52

Reply:
It's merged. Please add a separate issue for this.